source/game/kart/KartMove.hh

```diff
@@ -204,6 +204,7 @@ protected:
     f32 m_hopGravity; ///< Always main gravity (-1.3f).
     DrivingDirection m_drivingDirection; ///< Current state of driver's direction.
     s16 m_backwardsAllowCounter;         ///< Tracks the 15f delay before reversing.
+    bool m_bLaunchBoost;     ///< 不知道
```

Comment:
Note that this fails clang-format. It's not caught by our build actions here...

Reply:
You can resolve this by deleting that comment outright.